nautobot/extras/tables.py

```diff
@@ -493,3 +494,36 @@ class Meta(BaseTable.Meta):
             "content_type",
             "description",
         )
+
+
+class CustomFieldTable(BaseTable):
+    pk = ToggleColumn()
+    name = tables.Column(linkify=True, verbose_name="Slug")
+    content_types = ContentTypesColumn(truncate_words=15)
+    required = BooleanColumn()
+
+    class Meta(BaseTable.Meta):
+        model = CustomField
+        fields = (
+            "pk",
+            "name",
+            "content_types",
+            "type",
+            "label",
+            "description",
+            "required",
+            "default",
+            "weight",
+            "validation_minimum",
+            "validation_maximum",
+            "validation_regex",
```

Comment:
I'd consider removing the `validation` columns as being not generally relevant to a list view, though I'm open to persuasion on this point.